doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst

```diff
@@ -102,18 +102,26 @@ Intercomparison Project (LUMIP), which is part of CMIP6 Land use timeseries
 (:ref:`Lawrence et al. 2016 <Lawrenceetal2016>`). For more details about how
 crop distributions are determined, see Chapter :numref:`rst_Transient Landcover Change`. 
 
-CLM5 includes eight actively managed crop types
+CLM5 includes ten actively managed crop types
 (temperate soybean, tropical soybean, temperate corn, tropical 
-corn, spring wheat, cotton, rice, and sugarcane) that are chosen 
+corn, spring wheat, cotton, rice, sugarcane, miscanthus, and switchgrass) that are chosen 
 based on the availability of corresponding algorithms in AgroIBIS and as 
 developed by :ref:`Badger and Dirmeyer (2015)<BadgerandDirmeyer2015>` and
-described by :ref:`Levis et al. (2016)<Levisetal2016>`. The representations of
-sugarcane, rice, cotton, tropical corn, and tropical soy are new in CLM5.
+described by :ref:`Levis et al. (2016)<Levisetal2016>` and :ref:`Cheng et al. (2019)<Chengetal2019>`. 
+The representations of sugarcane, rice, cotton, tropical corn, tropical soy, miscanthus, and switchgrass are new in CLM5.
```

Comment:
Can you adjust the text to state that miscanthus and switchgrass were added after the CLM5 release?